.travis.yml

```diff
@@ -46,9 +46,6 @@ jobs:
     - <<: *node
       node_js: '10'
 
-    - <<: *node
-      node_js: '8'
-
     - script: npm start test.bundle test.browser
       # XXX: update when canvas supplies a prebuilt binary for Node.js v12.x
       node_js: 10
```

Comment:
Please check wether we can use `node_js: 12` now.

Reply:
Hi @juergba! I tried in a separate branch in my fork, but didn't work neither. https://github.com/UlisesGascon/mocha/pull/1/commits/67abb2c155e49b1116db229073d4b2121f76da6d. Not sure what is wrong on the CI 🤔. Any suggestion?

Reply:
mm... seems like Travis is now fine. Maybe I can add this UlisesGascon@67abb2c change to this PR. What do you think?

Reply:
Nothing has changed here, it's still set to `node_js: 10`. Why don't you just test it, and then we will know?

Reply:
This seems to be a more complex dependency problem. So we leave it unchanged for now.